sync_issues_to_jira/sync_issue.py

```diff
@@ -176,6 +183,29 @@ def handle_comment_deleted(jira, event):
     jira.add_comment(jira_issue.id, "@%s deleted [GitHub issue comment|%s]" % (gh_comment["user"]["login"], gh_comment["html_url"]))
 
 
+def sync_remain_prs(jira):
```

Comment:
As this is separate logic, what about making a `sync_issues_to_jira/sync_prs.py` file with the implementation? Can still import `sync_issue` if we need to use any data from there.